src/client/menu/menu.c

```diff
@@ -569,95 +607,144 @@ M_Main_Draw(void)
     {
         Draw_GetPicSize(&w, &h, names[i]);
 
-        if (w > widest)
-        {
+        if (w > widest) {
             widest = w;
```

Comment:
could we use origin code style?

Reply:
> looks as only code style change, could code have original code style?

*coug* *cough* i was wondering if you could have some spare time to review my pull request XD

Reply:
I may have retyped that. Yes code could have original code style. Does Yamagi have a code style convention? Are submissions required to follow the original id software code style convention?

Reply:
@Dremor8484 Your pull request won't go unnoticed, could you keep any comments related to this commits conversation. :)